app/src/main/java/com/example/smartshopper/services/RTDBService.java

```diff
@@ -94,19 +110,64 @@ public void writeComment(Comment comment, String dealID) {
         database.getReference().child(Constants.DEALS).child(dealID).child(Constants.COMMENTS).push().setValue(comment);
     }
 
-    public void upVoteDeal(String dealId) {
-        database.getReference()
-                .child(Constants.DEALS)
-                .child(dealId)
-                .child(Constants.UPVOTES)
-                .setValue(ServerValue.increment(1));
-    }
 
-    public void downVoteDeal(String dealId) {
```

Comment:
this seems to have been dropped from the merge. this needs to be in place for upvoting functions.